includes/widgets/icon.php

```diff
@@ -437,7 +437,7 @@ protected function render() {
 			$icon_tag = 'a';
 		}
 
-		if ( empty( $settings['icon'] ) && ! Icons_Manager::is_migration_allowed() ) {
+		if ( empty( $settings['icon'] ) && Icons_Manager::is_migration_required() ) {
```

Comment:
Maybe extract to a variable like in the other widgets?

Reply:
Why if it's only used once?

Reply:
Don't U use it in row 451?

Reply:
Right, changed